File: solutions/2048-build-array-from-permutation/solution.c
/**
 * Note: The returned array must be malloced, assume caller calls free().
 */
int* buildArray(int* nums, int numsSize, int* returnSize) {
    *returnSize=numsSize;
    int *nums1=(int *)malloc(4*numsSize);
    for(int i=0;i<numsSize;i++)
        nums1[i]=nums[nums[i]];
    return nums1;
}

File: solutions/2048-build-array-from-permutation/solution.py
class Solution(object):
    def buildArray(self, nums):
        """
        :type nums: List[int]
        :rtype: List[int]
        """
        nums1=[0]*len(nums)
        for i in range(len(nums)):
            nums1[i]=nums[nums[i]]
        return nums1
